subprojects/docs/src/snippets/maven-publish/specify-relocation/tests/publish-specify-relocation.sample.conf

```diff
@@ -0,0 +1,2 @@
+executable: gradle
+args: publish
```

Comment:
Is this configuration (and the snippets) reasonable? I copied them from [`subprojects/docs/src/snippets/ivy-publish/customize-identity`](https://github.com/gradle/gradle/tree/master/subprojects/docs/src/snippets/ivy-publish/customize-identity) and adjusted them.

Reply:
This looks reasonable.